tcms/templates/setup.html

```diff
@@ -0,0 +1,50 @@
+{% extends "base.html" %}
+{% load i18n %}
+{% load static %}
+
+{% block title %}{% trans "Setup - Migrations" %}{% endblock %}
+{% block body_class %}cards-pf{% endblock %}
+
+{% block contents %}
+    <div class="container-fluid container-cards-pf">
+        <div class="row row-cards-pf">
+            <div class="col-sm-12 col-md-12 col-lg-12">
+                <div class="card-pf card-pf-accented card-pf-aggregate-status card-pf-aggregate-status-mini">
+                    <h2 class="card-pf-title">
+                        <span class="fa fa-paper-plane"></span>
+                        <span class="card-pf-aggregate-status-count">
+                            Migrations
+                        </span>
+                        <br>
+                        Your database may need important migrations to be applied!
```

Comment:
Your database has not been initialized yet. Click the button below to initialize it!